src/api-adapter/react.js

```diff
@@ -21,54 +20,50 @@ function unwrap(value, getDevInfo) {
 }
 
 function getReactElementByNode(node) {
-  var element = elementMap[getID(node)];
-
-  if (element && getNode(element._rootNodeID)) {
-    return element;
+  if (node) {
+    return getElement(node);
   }
 
   return null;
 }
 
-function isComponentRootNode(node){
+function isComponentRootNode(node) {
   var element = getReactElementByNode(node);
-
-  if (element &&
-      element._currentElement &&
-      typeof element._currentElement != 'number' &&
-      typeof element._currentElement != 'string' &&
-      typeof element._currentElement.type != 'string') {
+  if (element && element._currentElement && typeof element._currentElement != "number" && typeof element._currentElement != "string") {
```

Comment:
I see only `typeof element._currentElement.type != 'string'` is removed. May be revert it with no one condition only?